src/Sylius/Component/Product/Command/UpdateBatchedVariants.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Product\Command;
```

Comment:
I wonder if it shouldn't be in Core as this command is specific to catalog promotions and has no sense only in Product component 🤔